berserker-http/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>berserker</artifactId>
+        <groupId>io.smartcat</groupId>
+        <version>0.0.8-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>berserker-http</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Berserker HTTP</name>
+    <description>Worker implementation which sends HTTP requests.</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <version.httpclient>4.5.3</version.httpclient>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.smartcat</groupId>
+            <artifactId>berserker-commons</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
What library does gatling use and how does it construct http requests?

Reply:
Gatling is using https://github.com/AsyncHttpClient/async-http-client
which is built on top of Netty. At first glance, it looks promising. I'd try it, maybe do some quick performance testing in comparison to one that's already included.